spec/compiler/crystal/tools/init_spec.cr

```diff
@@ -39,6 +39,10 @@ private def run_init_project(skeleton_type, name, author, email, github_name, di
   ).run
 end
 
+private def yaml(value)
+  YAML::Any.new(value)
```

Comment:
I'm not sure this extraction is worth it.

Reply:
Maybe not. I mainly did it to make the code that uses it less cluttered, not for DRY purposes. To be fair, I'm not sure how much better it is.

```crystal
parsed["targets"].should eq({
  yaml("example_app") => {
    yaml("main") => "src/example_app.cr",
  },
})

# vs

parsed["targets"].should eq({
  YAML::Any.new("example_app") => {
    YAML::Any.new("main") => "src/example_app.cr",
  },
})
```